Sources/MapboxNavigation/Bundle.swift

```diff
@@ -17,15 +17,13 @@ extension Bundle {
         }
     }
     
+    /**
+     Returns `UIImage` by searching for it in the current `Bundle` instance.
+     
+     - parameter named: Name of the image.
+     - returns: `UIImage` instance if image was found, `nil` otherwise.
+     */
     func image(named: String) -> UIImage? {
         return UIImage(named: named, in: self, compatibleWith: nil)
     }
-    
-    var microphoneUsageDescription: String? {
```

Comment:
Doesn't seem to be used anywhere. Removing it.

Reply:
👍 This was used for the audio feedback feature that was added in #719 but removed in #870.